src/drivers/gitlab.js

```diff
@@ -149,13 +171,18 @@ class Gitlab {
   }
 
   async request(opts = {}) {
-    const { token, api_v4 } = this;
+    const { token } = this;
     const { endpoint, method = 'GET', body, raw } = opts;
+    let { url } = opts;
 
-    if (!endpoint) throw new Error('Gitlab API endpoint not found');
+    if (endpoint) {
```

Comment:
```js
if (endpoint) {
      this.api_v4 = this.api_v4 || (await this.detect_api_v4());
      url = `${this.api_v4}${endpoint}`;
    } 

if (!url) throw new Error('Gitlab API url not found');
```

Reply:
dryer and is a validation in any circumstance

Reply:
@0x2b3bfa0

Reply:
👌 🙏